kubernetes-client/src/main/java/io/fabric8/kubernetes/client/informers/cache/ProcessorListener.java

```diff
@@ -60,7 +60,7 @@ public void run() {
         Thread.currentThread().interrupt();
         return;
       } catch (Exception ex) {
-        log.error("Failed invoking {} event handler: {}", ex.getMessage());
+        log.error("Failed invoking {} event handler: {}", handler, ex.getMessage(), ex);
```

Comment:
Logger.error only takes the Throwable without format

https://javadoc.io/static/org.slf4j/slf4j-api/1.7.30/org/slf4j/Logger.html#error-java.lang.String-java.lang.Throwable-

Reply:
I cannot find a documentation of this behaviour, but if the last argument of a format is an exception, then the exception is properly propataged to the underlying logger, so the associated stack trace may be printed.
For instance, see https://github.com/qos-ch/slf4j/blob/master/slf4j-api/src/main/java/org/slf4j/helpers/MessageFormatter.java#L415

Reply:
Do we want to log the stack trace though?

Reply:
I think we do. The exception is an error coming from user's code, the implementation of the event handler. As the implementer, I would like to know exactly where is the mistake I made. A simple message won't help me (don't blame me but some of my code still throws NPE from time to time 😅)